src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -116,9 +146,17 @@ define(function (require, exports, module) {
         rewriter.elements('video', 'src');
         rewriter.elements('audio', 'src');
 
+        rewriter.parseScripts('script','src');
```

Comment:
ye, i removed the args from the function itself, but forgot there, fixing coming